generic/tests/cfg/kdump.cfg

```diff
@@ -29,6 +29,8 @@
         kdump_propagate_cmd = "kdumpctl propagate"
         kdump_restart_cmd = "systemctl restart kdump.service"
 
+    auth_key_path = '/root/.ssh/authorized_keys'
+    authorized_key_cmd = "/bin/echo -e "%s" >> ${auth_key_path}"
```

Comment:
Other test cases do not really needed, we might skip the related steps when the parameters are not found.

Reply:
Okay,thanks for your advice.